tests/lib/AccountStoreIndexeddb.spec.js

```diff
@@ -0,0 +1,25 @@
+describe('AccountStore', () => {
+    it('is a singleton', () => {
+        const instance1 = AccountStore.instance;
+        const instance2 = AccountStore.instance;
+        expect(instance1).toBe(instance2);
+    });
+
+    it('connection can be opened and closed', async () => {
+        const db = await AccountStore.instance.connect();
+        expect(AccountStore.instance._dbPromise).toBeTruthy();
```

Comment:
Instead of using `AccountStore.instance` everywhere, you could store it in an `accountStore` variable instead.

Reply:
But using `.instance` better resembles how the AccountStore / KeyStore are used in the keyguard